curation_concerns-models/app/models/concerns/curation_concerns/upload_set_behavior.rb

```diff
@@ -7,7 +7,6 @@ module UploadSetBehavior
     included do
       has_many :file_sets, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf
 
-      property :creator, predicate: ::RDF::DC.creator
```

Comment:
We can *kinda* infer ownership through the `belongs_to` relationship declared on FileSets, though proxy depositor/transfer functionality could introduce inconsistencies. If I deposit 10 files and then transfer n of them to someone else at some point, who gets to perform operations on the UploadSet? Me or someone else? Part of this is that I think having the UploadSets persist in the repository is a potential place to hang other functionality in the future. Perhaps that is YAGNI territory, and we could put a flag in this for figuring out at that point.